rpm/kubelet.spec

```diff
@@ -41,7 +41,7 @@ Command-line utility for interacting with a Kubernetes cluster.
 %package -n kubeadm
 
 Version: 1.5.0
-Release: %{RPM_RELEASE}.alpha.0.1403.gc19e08e
+Release: %{RPM_RELEASE}.alpha.0.1534.gcf7301f
```

Comment:
What is 1534 here? Is it an int that will only ever increase? What is the 0 in front of it?

Technically we should probably bump the RPM_RELEASE above to 1, but given the way this is it's almost not necessary as you're updating something that should cause it to pick up as an update.

I'd bump RPM_RELEASE just to be safe and proper though.